src/harness/testcases/README.md

```diff
@@ -4,3 +4,39 @@
 
 The current test cases are based on the [WinnForum test specification](https://workspace.winnforum.org/higherlogic/ws/public/download/5637/WINNF-TS-0061-V1.1.0%20-%20WG4%20SAS%20Test%20and%20Certification%20Spec.pdf) v1.1 (Jan 2018)
 
+## Security Test Cases
+
+The SCS.11/16, SDS.11/16 and SSS.11/16 TCs just expect the handshake failure and do not validate the reason for failure.<br>
+The correct way to run the fake_sas for these test cases is
+<pre>
+<code>
+<b> $python fake_sas.py --verify_crl --ca ca_crl_chain.cert </b>
+</code>
+</pre>
+
+To obtain the CRL file, the <b>simple CRL server</b> in <b>master</b> branch needs to be 
+executed and a CRL chain can be retrieved from this server.
+
+The steps are
+
+1. Run the ‘simple CRL server’
+<b> $ python simple_crl_server.py </b>
+2. Use the menu option to blacklist a certificate (client cert or DP cert or SAS cert).
+3. Use wget on the simple CRL server URL to get the CRL chain file.
+<b> wget http://localhost:9007/ca.crl </b>
+4. Concatenate the CA chain file and the CRL file to create a combined CA chain and CRL chain file.
+<b> cat certs/ca.cert ca.crl > ca_crl_chain.cert </b>
```

Comment:
This step is incompatible with the next line.  `ca_crl_chain.cert` must be in the `certs/` directory.

Reply:
Agreed .   Updated accordingly.